assets/data/effects/drauven/drauven_pc_preparedShot.json

```diff
@@ -0,0 +1,68 @@
+{
+"id": "drauven_pc_preparedShot",
+"info": {
+	"dataVersion": 1,
+	"sourceFile": "druaven/drauven_pc_preparedShot",
+	"author": "Bipen",
+	"tags": [ "sneaky" ]
+},
+"type": "ABILITY",
+"verb": "ATTACK",
+"ability": {
+	"icon": "ambush",
+	"priority": "1",
+	"showDescriptionInTooltip": true,
+},
+"targets": [
+	{
+		"template": "SELF",
+		"missionFeedback": { "orientTo": "focus" }
+	},
+	{
+		"template": "TILE",
+		"alwaysSelect": true,
+		"missionFeedback": { "suggestionFeedback": "FRIENDLY_EFFECT_RANGE" },
+		"aiHints": {
+			"scoreNearbyFoe": "100",
+			"foeRange": "2",
+			"foeMaxRange": "10",
+			"scoreNearbyFriend": "10",
+			"friendRange": "2"
+		},
+		"minRange": "0.9",
+		"range": "(5.6-1)+self.BONUS_RANGE",
```

Comment:
Isn't this just equivalent to `4.6 + self.BONUS_RANGE`? What does the `-1` represent?

Reply:
need that -1 or wont work